v3/src/components/map/components/inspector-panel/map-base-layer-control.tsx

```diff
@@ -18,7 +18,8 @@ export const MapBaseLayerControl = observer(function MapBaseLayerControl(
   if (!mapModel) return null
 
   const classNameForSegment = (layerName: string) => {
-    return mapModel.baseMapLayerName === layerName ? 'map-base-button-selected' : 'map-base-button'
+    const baseClass = `map-base-button ${layerName}`
+    return mapModel.baseMapLayerName === layerName ?`${baseClass} selected` : baseClass
```

Comment:
I think this can be simplified to:
```suggestion
    return clsx("map-base-button", layerName, { selected: layerName === mapModel.baseMapLayerName })
```